### PR TITLE
Improve documentation 

### PR DESCRIPTION
**Pull request by pyhys**

### CODE REVIEW

File: doc/compliancetest.rst [hunk 1]
```diff
@@ -71,8 +50,10 @@ settings of your PC. In the properties for the driver, go to the "Advanced"
 tab. For the "Packet priority & VLAN" select "Packet priority & VLAN Disable".
 More details are given in the "Product Documentation" document for the tool.
 
-You might also need to turn off LLDP protocol for the selected network
-interface. Both Windows and Simatic TIA can have LLDP implemented.
+You also need to turn off LLDP protocol for the selected network interface,
```
Comment (olbjo) [inline doc/compliancetest.rst:53]: Clarify that this applies for test pc/station.

File: doc/multiple_ports.rst [hunk 2]
```diff
@@ -59,55 +59,89 @@ Enable and start network daemon::
...
-        inet 127.0.0.1  netmask 255.0.0.0
-        inet6 ::1  prefixlen 128  scopeid 0x10<host>
-        loop  txqueuelen 1000  (Local Loopback)
-        RX packets 0  bytes 0 (0.0 B)
-        RX errors 0  dropped 0  overruns 0  frame 0
-        TX packets 0  bytes 0 (0.0 B)
-        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
-
-Configuration of p-net stack and sample application 
+The sample app will set the ``br0`` bridge state to ``UP`` when started.
```
Comment (olbjo) [inline doc/multiple_ports.rst:62]: Is this true? Is the application controlling the network interfaces or just using them

**Review by olbjo — APPROVED** [review]

File: doc/multiple_ports.rst (hunk 2 above)
Comment (pyhys) [reply doc/multiple_ports.rst:62]: That is what happened when I tried.

File: doc/compliancetest.rst (hunk 1 above)
Comment (pyhys) [reply doc/compliancetest.rst:53]: Done

File: doc/multiple_ports.rst (hunk 2 above)
Comment (pyhys) [reply doc/multiple_ports.rst]: Modified the text. Added a separate file to bring the bridge up. Added paragraph on packet forwarding via bridge.
Done.

File: doc/getting_started_rtkernel.rst [hunk 3]
```diff
@@ -30,6 +30,10 @@ Run (adapt paths)::
 
     cp -r /opt/rt-tools/rt-kernel-xmc4 /home/jonas/projects/profinetstack/rtkernelcopy/
 
+Check out the relevant tag of the rt-kernel repository, for example::
```
[the remark this answers is not in the document]
Comment (pyhys) [reply doc/getting_started_rtkernel.rst]: Updated. 
Please check the new wording.

File: src/common/pf_lldp.c [hunk 4]
```diff
@@ -1028,12 +1028,12 @@ size_t pf_lldp_construct_frame (pnet_t * net, int loc_port_num, uint8_t buf[])
       "IP: %s Chassis ID: \"%s\" Port ID: \"%s\"\n",
       __LINE__,
       loc_port_num,
-      device_mac_address->addr[0],
-      device_mac_address->addr[1],
-      device_mac_address->addr[2],
-      device_mac_address->addr[3],
-      device_mac_address->addr[4],
-      device_mac_address->addr[5],
+      p_port_data->netif.mac_address.addr[0],
```
[the remark this answers is not in the document]
Comment (pyhys) [reply src/common/pf_lldp.c:1031]: Yes. This is for a debug log to show correct MAC address.

**Pull request merged**